testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MainActivity.java

```diff
@@ -269,21 +269,11 @@ public PublicClientApplication getPublicClientApplication() {
     public void onAcquireTokenSilentClicked(final AcquireTokenFragment.RequestOptions requestOptions) {
         prepareRequestParameters(requestOptions);
 
-        //final IAccount requestAccount = getAccount();
-        mApplication.getAccounts(new PublicClientApplication.AccountsLoadedCallback() {
+        mApplication.getAccount(requestOptions.getLoginHint().trim(), new PublicClientApplication.AccountsLoadedCallback<IAccount>() {
```

Comment:
Why is the `loginHint` being passed here?

Reply:
This is the loginhint text box in our test app. We can pass in the oid for testing here. I will add a comment there.